frontend/src/components/OrderList/data.js

```diff
@@ -0,0 +1,178 @@
+export const orders =[
+    {
```

Comment:
É importante lembrar de seguir esse mesmo padrão de resposta quando isso for implementado no backend.